tools/cockpit.spec

```diff
@@ -369,6 +369,11 @@ Recommends: setroubleshoot-server >= 3.3.3
 Provides: cockpit-selinux = %{version}-%{release}
 Provides: cockpit-sosreport = %{version}-%{release}
 %endif
+%if 0%{?fedora} >= 29
+# 0.7.0 (actually) supports task cancellation.
+# 0.7.1 fixes tasks never announcing completion.
+Recommends: (reportd >= 0.7.1 if abrt)
```

Comment:
Is this ever aimed at RHEL as well, or will this stay a Fedora-ism for the time being?

Reply:
For the time being, it will stay a Fedora-ism. The closest to RHEL is the EPEL 8 branch I have, but I’ll need to maintain a separate codebase.